test/integ/test_large_put.py

```diff
@@ -51,8 +51,10 @@ def test_put_copy_large_files(tmpdir, conn_cnx, db_parameters):
             files = files.replace("\\", "\\\\")
             cnx.cursor().execute(
                 "put 'file://{file}' @%{name}".format(
-                    file=files, name=db_parameters["name"]
-                )
+                    file=files,
+                    name=db_parameters["name"],
+                ),
+                _multipart_threshold=1000000,
```

Comment:
Does this change force to use multipart upload or other way around? Sorry I'm not up to date with the latest Python Connector. It would be great if you can add comments here about your intention.

Reply:
Comment added. If client provide this threshold, client wont use the threshold provided in the server response to PUT request.